### PR TITLE
fix(baseline): remove data loader from arima class

### PR DESCRIPTION
**Pull request by eduardocorrearaujo**
Solve issue #16 
@fccoelho I made the changes that you requested. But, I am not sure that the usability is good, and I would like some feedback. Look at the notebook `baseline_model.ipynb`. Lua is solving the warnings showing in the import.

### CODE REVIEW

File: examples/baseline_model.ipynb [hunk 1]
```diff
@@ -2,20 +2,158 @@
...
+    "geocode = 3304557 \n",
+    "end_date = date.today().strftime('%Y-%m-%d')\n",
+    "\n",
+    "df = Infodengue.get(per_page =  100,\n",
+    "                            disease =  disease,\n",
+    "                            start = \"2010-01-01\",\n",
+    "                            end = end_date, \n",
+    "                            geocode = geocode)\n",
+    "\n",
+    "df.set_index('data_iniSE', inplace = True )\n",
```
Comment (fccoelho) [inline examples/baseline_model.ipynb:136]: Normally I  convert to datetime before setting a column as  index. There is no need to sort,  Datetime index already has an inherent order.

File: examples/baseline_model.ipynb [hunk 2]
```diff
@@ -2,20 +2,158 @@
...
+    "    \n",
+    "df.index = pd.to_datetime(df.index)\n",
+    "    \n",
+    "df = df.sort_index()\n",
+    "    \n",
+    "df = df.asfreq('W')\n",
```
Comment (fccoelho) [inline examples/baseline_model.ipynb:142]: Here I think it is better to use `df = df.resample('W').sum()` there is no need to `ffill` afterwards.

**Review by fccoelho — CHANGES_REQUESTED** [review]

**Comment by fccoelho** [comment]
@eduardocorrearaujo, It is better now, because it is easier for the user to know ho to fetch the data from the API.
Just make these small changes I ask to make the code a bit shorter and clearer.

**Comment by eduardocorrearaujo** [comment]
> @eduardocorrearaujo, It is better now, because it is easier for the user to know ho to fetch the data from the API. Just make these small changes I ask to make the code a bit shorter and clearer.

Done!

**Pull request merged**